esx_service/vmci/vmci_client.c

```diff
@@ -161,9 +165,10 @@ dummy_get_reply(be_sock_id *id, be_request *r, be_answer* a)
 static be_sock_status
 vsock_init(be_sock_id *id, int cid, int port)
```

Comment:
That is the port for the ESX service which we need to be configurable and is based on config in service file. We initially used the same port on the client side for simplicity.